ehitchhiking-ui/src/app/pages/main-screen/routes-list/routes-list.component.sass

```diff
@@ -0,0 +1,32 @@
+.routes-list
+  z-index: 1
+  background-color: white
+  height: 22.5rem
+  width: 26rem
+  box-shadow: 0 0 9px rgba(0, 0, 0, 0.2)
+  border-radius: 0.2rem
+  position: absolute
+  margin-left: -200px
+  bottom: 4rem
+  left: 50%
+  position: fixed
+  overflow-y: auto
+  .routes-list-p
+    display: flex
+    justify-content: space-between
+  .route-info
+    margin: 1rem
+    .material-icons
+     font-size: 2rem
+  .right-span
+    text-align: center
+    display: flex
+    align-items: center
+    justify-content: center
+.header
+  height: 5rem!important
```

Comment:
pls add comment why !important is required

Reply:
fixed